vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLMessage.java

```diff
@@ -0,0 +1,52 @@
+package io.vertx.ext.web.handler.graphql.impl;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class GraphQLMessage {
+
+  private String id;
+
+  private Type type;
+
+  public String getId() {
```

Comment:
Can you rename the Java field to `opId` for clarity and use Jackson annotations to keep the JSON `id` field name?